axelrod/tests/unit/test_tournament.py

```diff
@@ -745,6 +731,55 @@ def test_write_to_csv_without_results(self):
         expected_df = pd.read_csv(axl_filename(path))
         self.assertTrue(df.equals(expected_df))
 
+    @given(seed=integers(min_value=1, max_value=4294967295))
+    @settings(max_examples=5, deadline=None)
+    def test_seeding_equality(self, seed):
+        """Tests that a tournament with a given seed will return the
+        same results each time."""
+        rng = axl.RandomGenerator(seed=seed)
+        players = [axl.Random(rng.random()) for _ in range(8)]
+        tournament1 = axl.Tournament(
+            name=self.test_name,
+            players=players,
+            game=self.game,
+            turns=3,
+            repetitions=100,
+            seed=seed
+        )
+        tournament2 = axl.Tournament(
+            name=self.test_name,
+            players=players,
+            game=self.game,
+            turns=3,
+            repetitions=100,
+            seed=seed
+        )
+        results1 = tournament1.play(processes=2)
+        results2 = tournament2.play(processes=2)
+        self.assertEqual(results1.ranked_names, results2.ranked_names)
```

Comment:
Could we pop this in a loop so as to repeat the assertion a few more times? Or perhaps we run 10 tournaments and not just 2?

Reply:
Yeah we could do something like that, or make better use of hypothesis here. I'm open to suggestions, but we also probably want to be careful to avoid a long runtime test.

Reply:
Yeah runtime is vital. I'd suggest just throwing it in a loop would be sufficient.

```
for _ in range(4):
     ...
```

Is all I'm thinking.